data/json/monsters.json

```diff
@@ -2510,6 +2549,37 @@
     "flags":["SEES", "HEARS", "SMELLS", "WARM", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "BONES"]
     },{
     "type" : "MONSTER",
+    "id" : "mon_zombie_predator",
+    "name": "feral predator",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"brown_green",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":6,
+    "aggression":100,
+    "morale":100,
+    "speed":170,
```

Comment:
It has enough health to survive being shot at at range, unless something stuns it. Test it without an armor (maybe even as nudist) - it should be killable even after it manages to land the first impale+infection.
As much as I'm for difficulty, this one is a huge difficulty spike. It's basically a jumping hulk that sees in the darkness and can be only escaped from in the city (by making it lose the line of sight) and smashes you into the ground rather than away from itself.